### PR TITLE
Add Manibus keyboard and layout from Blank Tehnologii

### PR DESCRIPTION
**Pull request by SamuraiKek**
## Description
Initial PR adding Manibus keyboard. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/blank_tehnologii/manibus/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+#include QMK_KEYBOARD_H
+
+
+#define _QWERTY 0
+#define _LOWER 1
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER
+};
```
Comment (drashna) [inline keyboards/blank_tehnologii/manibus/keymaps/default/keymap.c:25]:
```suggestion
enum custom_keycodes {
  _QWERTY,
  _LOWER
};

#define LOWER MO(_LOWER)
```

File: keyboards/blank_tehnologii/manibus/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    case LOWER:
+      if (record->event.pressed) {
+        layer_on(_LOWER);
+      } else {
+        layer_off(_LOWER);
+      }
+      return false;
+  }
+  return true;
+}
```
Comment (drashna) [inline keyboards/blank_tehnologii/manibus/keymaps/default/keymap.c:96]:
```suggestion
```

File: keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _UPPER 2
+#define _MOUSE 3
+#define _MEDIA 4
```
Comment (drashna) [inline keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c:23]: We highly recommend using enums for the layer naming, as it is easier to maintain long term. 

```suggestion
enum custom_layers {
    _QWERTY,
    _LOWER,
    _UPPER,
    _MOUSE,
    _MEDIA,  
);
```

File: keyboards/blank_tehnologii/manibus/rules.mk [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
```
Comment (drashna) [inline keyboards/blank_tehnologii/manibus/rules.mk:59]:
```suggestion
MCU = atmega32u4

# Bootloader selection
BOOTLOADER = atmel-dfu
```

File: keyboards/blank_tehnologii/manibus/rules.mk [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
```
Comment (drashna) [inline keyboards/blank_tehnologii/manibus/rules.mk:81]:
```suggestion
# Build Options
#   change yes to no to disable
BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
UNICODE_ENABLE = no         # Unicode
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/blank_tehnologii/manibus/readme.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+Keyboard Maintainer: [Alex Paulescu/SamuraiKek](https://github.com/SamuraiKek)
+
+Hardware Supported: [ATmega32u4](https://www.digikey.com/en/products/detail/microchip-technology/ATMEGA32U4-AU/1914602)
+
+Hardware Availability: [Blank Tehnologii](https://blank.computer/)
+
+Make example for this keyboard (after setting up your build environment):
+
+```make blank_tehnologii/manibus:default```
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/readme.md:15]:
```suggestion
    make blank_tehnologii/manibus:default
```

File: keyboards/blank_tehnologii/manibus/readme.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+Example of flashing this keyboard:
+
+Left half: 
+```make blank_tehnologii/manibus:default:dfu-split-left```
+
+Right half:
+```make blank_tehnologii/manibus:default:dfu-split-right```
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/readme.md:23]:
```suggestion
    make blank_tehnologii/manibus:default:dfu-split-left # Left half
    make blank_tehnologii/manibus:default:dfu-split-right # Right half
```

File: keyboards/blank_tehnologii/manibus/readme.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+See [build environment setup](https://docs.qmk.fm/#/newbs_getting_started?id=_5-configure-your-build-environment-optional) and [handedness by EEPROM](https://docs.qmk.fm/#/feature_split_keyboard?id=handedness-by-eeprom) for more information.
+
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/readme.md:26]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/blank_tehnologii/manibus/readme.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+![Imgur](https://i.imgur.com/0l17Xyy.jpg)
+
+A 5x7 split ergonomic keyboard designed and sold by [Blank Tehnologii](https://blank.computer/).
+
+Keyboard Maintainer: [Alex Paulescu/SamuraiKek](https://github.com/SamuraiKek)
+
+Hardware Supported: [ATmega32u4](https://www.digikey.com/en/products/detail/microchip-technology/ATMEGA32U4-AU/1914602)
+
+Hardware Availability: [Blank Tehnologii](https://blank.computer/)
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/readme.md:11]:
```suggestion
* Keyboard Maintainer: [Alex Paulescu/SamuraiKek](https://github.com/SamuraiKek)
* Hardware Supported: [ATmega32u4](https://www.digikey.com/en/products/detail/microchip-technology/ATMEGA32U4-AU/1914602)
*  Hardware Availability: [Blank Tehnologii](https://blank.computer/)
```

File: keyboards/blank_tehnologii/manibus/readme.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
+# Manibus keyboard
+
+![Imgur](https://i.imgur.com/0l17Xyy.jpg)
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/readme.md:3]:
```suggestion
![Imgur](https://i.imgur.com/0l17Xyyl.jpg)
```

File: keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+      _______, _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______, _______
+//   └────────┴────────┴────────┴────────┴────────┴────────┴────────┘     └────────┴────────┴────────┴────────┴────────┴────────┴────────┘
+  )
+};
+
+// Automatically sets the numlock on at startup
+void led_set_keymap(uint8_t usb_led) {
+  if (!(usb_led & (1<<USB_LED_NUM_LOCK))) {
+    register_code(KC_NUMLOCK);
+    unregister_code(KC_NUMLOCK);
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c:141]:
```suggestion
    tap_code(KC_NLCK);
```

File: keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+        }
+        return false;
+    case KC_CTLT:
+        if(record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LCTL)SS_TAP(X_T)SS_UP(X_LCTL));
+        }
+        return false;
+    case KC_CST:
+        if(record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LCTL)SS_DOWN(X_LSHIFT)SS_TAP(X_T)SS_UP(X_LCTL)SS_UP(X_LSHIFT));
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c:207]:
```suggestion
            tap_code16(C(S(KC_T)));
```

File: keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+      }
+      return false;
+    case KC_CTLW:
+        if(record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LCTL)SS_TAP(X_W)SS_UP(X_LCTL));
+        }
+        return false;
+    case KC_CTLT:
+        if(record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LCTL)SS_TAP(X_T)SS_UP(X_LCTL));
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c:202]:
```suggestion
            tap_code16(C(KC_T));
```

File: keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c [hunk 14]
```diff
@@ -0,0 +1,222 @@
...
+      }
+      return false;
+    case KC_QUIT:
+      if(record->event.pressed) {
+        SEND_STRING(SS_DOWN(X_LALT)SS_TAP(X_F4)SS_UP(X_LALT));
+      }
+      return false;
+    case KC_CTLW:
+        if(record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LCTL)SS_TAP(X_W)SS_UP(X_LCTL));
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c:197]:
```suggestion
            tap_code16(C(KC_W));
```

File: keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c [hunk 15]
```diff
@@ -0,0 +1,222 @@
...
+      }
+      return false;
+    case KC_CSTB:
+      if(record->event.pressed) {
+        SEND_STRING(SS_DOWN(X_LCTL)SS_DOWN(X_LSHIFT)SS_TAP(X_TAB)SS_UP(X_LCTL)SS_UP(X_LSHIFT));
+      }
+      return false;
+    case KC_QUIT:
+      if(record->event.pressed) {
+        SEND_STRING(SS_DOWN(X_LALT)SS_TAP(X_F4)SS_UP(X_LALT));
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c:192]:
```suggestion
        tap_code16(A(KC_F4));
```

File: keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c [hunk 16]
```diff
@@ -0,0 +1,222 @@
...
+      }
+      return false;
+    case KC_CTB:
+      if(record->event.pressed) {
+        SEND_STRING(SS_DOWN(X_LCTL)SS_TAP(X_TAB)SS_UP(X_LCTL));
+      }
+      return false;
+    case KC_CSTB:
+      if(record->event.pressed) {
+        SEND_STRING(SS_DOWN(X_LCTL)SS_DOWN(X_LSHIFT)SS_TAP(X_TAB)SS_UP(X_LCTL)SS_UP(X_LSHIFT));
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c:187]:
```suggestion
        tap_code16(C(S(KC_TAB)));
```

File: keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c [hunk 17]
```diff
@@ -0,0 +1,222 @@
...
+    case MEDIA:
+      if(record->event.pressed) {
+        layer_on(_MEDIA);
+      } else {
+        layer_off(_MEDIA);
+      }
+      return false;
+    case KC_CTB:
+      if(record->event.pressed) {
+        SEND_STRING(SS_DOWN(X_LCTL)SS_TAP(X_TAB)SS_UP(X_LCTL));
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c:182]:
```suggestion
        tap_code16(C(KC_TAB));
```

File: keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c [hunk 18]
```diff
@@ -0,0 +1,222 @@
...
+        return false;
+    case KC_CST:
+        if(record->event.pressed) {
+            SEND_STRING(SS_DOWN(X_LCTL)SS_DOWN(X_LSHIFT)SS_TAP(X_T)SS_UP(X_LCTL)SS_UP(X_LSHIFT));
+        }
+        return false;
+    case  KC_ENT:
+        if(record->event.pressed) {
+          if(get_mods() & MOD_BIT(KC_LSFT)){
+            SEND_STRING(SS_TAP(X_SPC));
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c:213]:
```suggestion
            tap_code(KC_SPC);
```

File: keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c [hunk 19]
```diff
@@ -0,0 +1,222 @@
...
+          }
+          else {
+             SEND_STRING(SS_TAP(X_ENT));
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c:216]:
```suggestion
             tap_code(KC_ENT);
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by SamuraiKek** [comment]
Thanks for the changes folks. Some nice improvements to the code that I didn't know about :D

**Review by drashna — APPROVED** [review]

File: keyboards/blank_tehnologii/manibus/rules.mk [hunk 20]
```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/rules.mk:7]:
```suggestion
```

File: keyboards/blank_tehnologii/manibus/readme.md [hunk 21]
```diff
@@ -0,0 +1,20 @@
+# Manibus keyboard
+
+![Imgur](https://i.imgur.com/0l17Xyyl.jpg)
+
+A 5x7 split ergonomic keyboard designed and sold by [Blank Tehnologii](https://blank.computer/).
+
+* Keyboard Maintainer: [Alex Paulescu/SamuraiKek](https://github.com/SamuraiKek)
+* Hardware Supported: [ATmega32u4](https://www.digikey.com/en/products/detail/microchip-technology/ATMEGA32U4-AU/1914602)
+*  Hardware Availability: [Blank Tehnologii](https://blank.computer/)
```
Comment (fauxpark) [inline keyboards/blank_tehnologii/manibus/readme.md:9]:
```suggestion
* Hardware Availability: [Blank Tehnologii](https://blank.computer/)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**